Mass Commit: Added all the tests

saquibnazmus opened the pull request:
Hello Reviewer, 

I have added all the tests. However, you will notice that, JSAlerts prompt test are not working as expected. The rest of the tests should be fine. If you have any questions please feel free to reach out to me. 

I look forward to hearing from you regarding any feedback you have.

Thanks,
-Nazmus

python-api-test commented:
I'm not able to execute tests on IntelliJ:

[AppClassLoader@2c13da15] warning javax.* types are not being woven because the weaver option '-Xset:weaveJavaxPackages=true' has not been specified

===============================================
Default Suite
Total tests run: 0, Failures: 0, Skips: 0
===============================================


Process finished with exit code 0

saquibnazmus commented:
Closing the PR since, no response form Recruiter.

hbahuguna2 commented:
> Closing the PR since, no response form Recruiter.

Could you please check the comments above(also posting here again) and provide with a fix or solution. 

I'm not able to execute tests on IntelliJ:

[AppClassLoader@2c13da15] warning javax.* types are not being woven because the weaver option '-Xset:weaveJavaxPackages=true' has not been specified

saquibnazmus commented:
Hi,

Apologies for closing the PR.

Could you please provide more details about the error?

   - Are Maven dependencies included in the project properly?
   - Are you able to see aspectjweaver JAR in the external libs?
   - Is "mvn clean compile" working?
   - Is it possible to set system properties as shown in the screenshot?
   (BaseTest.Java)
   - Is the project structure set to Java 1.8?

I am able to execute locally. Would you like to have a quick call to
go over this?

Let me know.

Thanks,
-Nazmus

On Tue, Nov 10, 2020 at 10:23 AM hbahuguna2 <notifications@github.com>
wrote:

> Closing the PR since, no response form Recruiter.
>
> Could you please check the comments above(also posting here again) and
> provide with a fix or solution.
>
> I'm not able to execute tests on IntelliJ:
>
> [AppClassLoader@2c13da15] warning javax.* types are not being woven
> because the weaver option '-Xset:weaveJavaxPackages=true' has not been
> specified
>
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly, view it on GitHub
> <https://github.com/Dotdashcom/take-home-webdriver-test/pull/82#issuecomment-724772824>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ARTMFJXISLIEWYEQ2CDRM33SPFLITANCNFSM4TMFJI4A>
> .
>


-- 

Nazmus Saquib

saquibnazmus commented:
Hi,

Also, I was wondering how you have set your JVM settings and may be
increase the Xms and Xmx settings.

Let me know if you are running the same issue.

Looking forward to hearing from you.

Thanks,
-Nazmus

On Tue, Nov 10, 2020 at 12:41 PM Nazmus Saquib <nazmus.saquib@gmail.com>
wrote:

> Hi,
>
> Apologies for closing the PR.
>
> Could you please provide more details about the error?
>
>    - Are Maven dependencies included in the project properly?
>    - Are you able to see aspectjweaver JAR in the external libs?
>    - Is "mvn clean compile" working?
>    - Is it possible to set system properties as shown in the screenshot?
>    (BaseTest.Java)
>    - Is the project structure set to Java 1.8?
>
> I am able to execute locally. Would you like to have a quick call to
> go over this?
>
> Let me know.
>
> Thanks,
> -Nazmus
>
> On Tue, Nov 10, 2020 at 10:23 AM hbahuguna2 <notifications@github.com>
> wrote:
>
>> Closing the PR since, no response form Recruiter.
>>
>> Could you please check the comments above(also posting here again) and
>> provide with a fix or solution.
>>
>> I'm not able to execute tests on IntelliJ:
>>
>> [AppClassLoader@2c13da15] warning javax.* types are not being woven
>> because the weaver option '-Xset:weaveJavaxPackages=true' has not been
>> specified
>>
>> —
>> You are receiving this because you modified the open/close state.
>> Reply to this email directly, view it on GitHub
>> <https://github.com/Dotdashcom/take-home-webdriver-test/pull/82#issuecomment-724772824>,
>> or unsubscribe
>> <https://github.com/notifications/unsubscribe-auth/ARTMFJXISLIEWYEQ2CDRM33SPFLITANCNFSM4TMFJI4A>
>> .
>>
>
>
> --
>
> Nazmus Saquib
>


-- 

Nazmus Saquib

hbahuguna2 commented:
> Hi, Apologies for closing the PR. Could you please provide more details about the error? - Are Maven dependencies included in the project properly? - Are you able to see aspectjweaver JAR in the external libs? - Is "mvn clean compile" working? - Is it possible to set system properties as shown in the screenshot? (BaseTest.Java) - Is the project structure set to Java 1.8? I am able to execute locally. Would you like to have a quick call to go over this? Let me know. Thanks, -Nazmus
> […](#)
> On Tue, Nov 10, 2020 at 10:23 AM hbahuguna2 ***@***.***> wrote: Closing the PR since, no response form Recruiter. Could you please check the comments above(also posting here again) and provide with a fix or solution. I'm not able to execute tests on IntelliJ: ***@***.*** warning javax.* types are not being woven because the weaver option '-Xset:weaveJavaxPackages=true' has not been specified — You are receiving this because you modified the open/close state. Reply to this email directly, view it on GitHub <[#82 (comment)](https://github.com/Dotdashcom/take-home-webdriver-test/pull/82#issuecomment-724772824)>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/ARTMFJXISLIEWYEQ2CDRM33SPFLITANCNFSM4TMFJI4A> .
> -- Nazmus Saquib
Below is the screenshot of my IntelliJ when I run `CheckBoxesTest` test(I get same result with all tests):

![image](https://user-images.githubusercontent.com/46648825/98715129-f630ac80-2346-11eb-817e-b6ed019e7f1d.png)

   - Are Maven dependencies included in the project properly?
      Yes, I have made no changes to POM and imported project in IntelliJ as it is.
   - Are you able to see aspectjweaver JAR in the external libs?
      I'm not able to see `aspectjweaver` in external libs, `aspectjrt` is available though. Why is `aspectjweaver` needed? Can we write tests without `aspectjweaver`?
![image](https://user-images.githubusercontent.com/46648825/98714416-1613a080-2346-11eb-9ac5-be5ca9c9d33c.png)

   - Is "mvn clean compile" working?
      Yes `mvn clean compile` is working.
   - Is it possible to set system properties as shown in the screenshot?
   (BaseTest.Java)
     Somehow I can't see this screenshot.
   - Is the project structure set to Java 1.8?
      yes.

hbahuguna2 commented:
> Hi, Also, I was wondering how you have set your JVM settings and may be increase the Xms and Xmx settings. Let me know if you are running the same issue. Looking forward to hearing from you. Thanks, -Nazmus On Tue, Nov 10, 2020 at 12:41 PM Nazmus Saquib <nazmus.saquib@gmail.com> wrote:
> […](#)
> Hi, Apologies for closing the PR. Could you please provide more details about the error? - Are Maven dependencies included in the project properly? - Are you able to see aspectjweaver JAR in the external libs? - Is "mvn clean compile" working? - Is it possible to set system properties as shown in the screenshot? (BaseTest.Java) - Is the project structure set to Java 1.8? I am able to execute locally. Would you like to have a quick call to go over this? Let me know. Thanks, -Nazmus On Tue, Nov 10, 2020 at 10:23 AM hbahuguna2 ***@***.***> wrote: > Closing the PR since, no response form Recruiter. > > Could you please check the comments above(also posting here again) and > provide with a fix or solution. > > I'm not able to execute tests on IntelliJ: > > ***@***.*** warning javax.* types are not being woven > because the weaver option '-Xset:weaveJavaxPackages=true' has not been > specified > > — > You are receiving this because you modified the open/close state. > Reply to this email directly, view it on GitHub > <[#82 (comment)](https://github.com/Dotdashcom/take-home-webdriver-test/pull/82#issuecomment-724772824)>, > or unsubscribe > <https://github.com/notifications/unsubscribe-auth/ARTMFJXISLIEWYEQ2CDRM33SPFLITANCNFSM4TMFJI4A> > . > -- Nazmus Saquib
> -- Nazmus Saquib

ok, I changed `<argLine>
                        -javaagent:"${settings.localRepository}/org/aspectj/aspectjweaver/${aspectj.version}/aspectjweaver-${aspectj.version}.jar"
                    </argLine>` to `<argLine>-Xms512m -Xmx1024m</argLine>` as per [this](https://stackoverflow.com/questions/36976374/warning-javax-types-are-not-being-woven-because-the-weaver-option-xsetweave) post, and it seems to be working now.

saquibnazmus commented:
Hi,

Correct, that was also one of my suspicions, that the JVM settings may need
to be increased.

I have implemented Allure previously and haven't seen that issue before.
The basic installation is provided here
<https://docs.qameta.io/allure/#_installation>.

Thanks,
-Nazmus



On Tue, Nov 10, 2020 at 1:36 PM hbahuguna2 <notifications@github.com> wrote:

> Hi, Also, I was wondering how you have set your JVM settings and may be
> increase the Xms and Xmx settings. Let me know if you are running the same
> issue. Looking forward to hearing from you. Thanks, -Nazmus On Tue, Nov 10,
> 2020 at 12:41 PM Nazmus Saquib nazmus.saquib@gmail.com wrote:
> … <#m_2878051544858819781_m_-4285690525088066892_>
> Hi, Apologies for closing the PR. Could you please provide more details
> about the error? - Are Maven dependencies included in the project properly?
> - Are you able to see aspectjweaver JAR in the external libs? - Is "mvn
> clean compile" working? - Is it possible to set system properties as shown
> in the screenshot? (BaseTest.Java) - Is the project structure set to Java
> 1.8? I am able to execute locally. Would you like to have a quick call to
> go over this? Let me know. Thanks, -Nazmus On Tue, Nov 10, 2020 at 10:23 AM
> hbahuguna2 *@*.*> wrote: > Closing the PR since, no response form
> Recruiter. > > Could you please check the comments above(also posting here
> again) and > provide with a fix or solution. > > I'm not able to execute
> tests on IntelliJ: > > @.* warning javax.* types are not being woven >
> because the weaver option '-Xset:weaveJavaxPackages=true' has not been >
> specified > > — > You are receiving this because you modified the
> open/close state. > Reply to this email directly, view it on GitHub > <#82
> (comment)
> <https://github.com/Dotdashcom/take-home-webdriver-test/pull/82#issuecomment-724772824>>,
> > or unsubscribe >
> https://github.com/notifications/unsubscribe-auth/ARTMFJXISLIEWYEQ2CDRM33SPFLITANCNFSM4TMFJI4A
> > . > -- Nazmus Saquib
> -- Nazmus Saquib
>
> ok, I changed <argLine>
> -javaagent:"${settings.localRepository}/org/aspectj/aspectjweaver/${aspectj.version}/aspectjweaver-${aspectj.version}.jar"
> </argLine> to <argLine>-Xms512m -Xmx1024m</argLine> as per this
> <https://stackoverflow.com/questions/36976374/warning-javax-types-are-not-being-woven-because-the-weaver-option-xsetweave>
> post, and it seems to be working now.
>
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly, view it on GitHub
> <https://github.com/Dotdashcom/take-home-webdriver-test/pull/82#issuecomment-724887752>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ARTMFJSVHLFRNKCDX7TUF3DSPGBZPANCNFSM4TMFJI4A>
> .
>
